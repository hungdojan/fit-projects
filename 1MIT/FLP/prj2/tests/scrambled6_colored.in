oyr
rww
gww
wbb oob wog wbr
ggg rry gbg woo
bby oby gry brr
yyg
yyo
owr
