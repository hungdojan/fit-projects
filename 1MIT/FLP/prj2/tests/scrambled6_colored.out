oyr
rww
gww
wbb oob wog wbr
ggg rry gbg woo
bby oby gry brr
yyg
yyo
owr

bww
rww
gww
wbb ooo ggy obr
ggg rry obr woo
bby obr wgg rrr
yyg
yyo
yyb

grb
www
www
ooo ggy obr wbb
ggg rry obr woo
bby obr wgg rrr
yyg
yyo
yyb

grb
www
www
ooo ggy obr wbb
ggg rry obr woo
rrr bby obr wgg
yyy
yyy
bog

www
www
www
ooo ggg rrr bbb
ggg rrr bbb ooo
rrr bbb ooo ggg
yyy
yyy
yyy

www
www
www
ggg rrr bbb ooo
ggg rrr bbb ooo
rrr bbb ooo ggg
yyy
yyy
yyy

www
www
www
ggg rrr bbb ooo
ggg rrr bbb ooo
ggg rrr bbb ooo
yyy
yyy
yyy
